src/app/YoutubeResponseInterface.ts

```diff
@@ -0,0 +1,61 @@
+export interface YtResponse {
+  kind: string;
+  etag: string;
+  items: Item[];
+  pageInfo: PageInfo;
+}
+
+export interface Item {
+  kind: string;
+  etag: string;
+  id: string;
+  snippet: Snippet;
+  statistics: Statistics;
+}
+
+export interface Snippet {
+  publishedAt: Date;
+  channelId: string;
+  title: string;
+  description: string;
+  thumbnails: Thumbnails;
+  channelTitle: string;
+  tags: string[];
+  categoryId: string;
+  liveBroadcastContent: string;
+  defaultLanguage: string;
+  localized: Localized;
+  defaultAudioLanguage: string;
+}
+
+export interface Localized {
+  title: string;
+  description: string;
+}
+
+export interface Thumbnails {
+  default: Default;
+  medium: Default;
+  high: Default;
+  standard: Default;
+  maxres: Default;
+}
+
+export interface Default {
```

Comment:
tu generator chujowo Ci nazwał interfejs - w 37 linii property nazywa się default i musiał po tym wnioskować. To chyba jakieś thumbnaile więc może Thumbnail
ogólnie warto przejrzeć takie automatycznie generowane rzeczy